pkg/plugins/user_test.go

```diff
@@ -94,33 +138,32 @@ last:x:999:999:Test user for uid:/:/usr/bin/nologin
 
 			shadow, err := fs.ReadFile("/etc/shadow")
 			Expect(err).ShouldNot(HaveOccurred())
-			//passwd, err := fs.ReadFile("/etc/passwd")
 			Expect(err).ShouldNot(HaveOccurred())
 			group, err := fs.ReadFile("/etc/group")
 			Expect(err).ShouldNot(HaveOccurred())
 
 			passdRaw, _ := fs.RawPath("/etc/passwd")
-			passwd, _ := passwd2.ParseFile(passdRaw)
 
-			checkDefaultUsers(passwd)
+			list := xpasswd.NewUserList()
+			list.SetPath(passdRaw)
+			list.Load()
+			Expect(list).To(HaveAllDefaultUsers())
 
 			Expect(string(group)).Should(Equal("foo:x:1000:foo\n"))
 
 			Expect(string(shadow)).Should(ContainSubstring("foo:$fkekofe:"))
-			Expect(passwd["foo"]).ToNot(BeNil())
-			foo := passwd["foo"]
+			foo := list.Get("foo")
 			Expect(foo).ToNot(BeNil())
-			Expect(foo.Gecos).To(Equal("Created by entities"))
-			Expect(foo.Pass).To(Equal("x"))
```

Comment:
aren't we testing Pass anymore? what's the default in this case?

Reply:
this is not a real pass test, there's always an x in `/etc/passwd` the real passwords are saved in the shadow file

Reply:
right, but it is worth checking if we write x here by default - to check that we are still compliant and we do not break while generating the user in the /etc/passwd file

Reply:
ahh, sorry I misunderstood the question. Yeah, I decided it's not a good API to ask xpasswd for a `.Pass()` because it's not representative, the `x` is not the password so if I implement that I want it to return the value in the shadow file.

I would claim that for this test it is actually ok because when we `Load()` the passwd file, `xpasswd` validates that the list fulfills the 7 elements per item. But it indeed does not check that it specifically is an `x` but as far as i understand the value there should not break the functionality

Reply:
I'll change to at least check that error, and will think of a better option

Reply:
@mudler I've updated it to also check the password